lib/osv/ecosystems_test.py

```diff
@@ -82,6 +82,20 @@ def test_nuget(self):
     with self.assertRaises(ecosystems.EnumerateError):
       ecosystem.next_version('doesnotexist123456', '1')
 
+  def test_debian(self):
```

Comment:
Thanks for adding this test! Could you also add a more end-to-end test here: https://github.com/google/osv/blob/367d977409547d62713d15b59dec50f0bf1228d5/docker/worker/worker_test.py#L930